book/src/SUMMARY.md

```diff
@@ -10,7 +10,11 @@
   - [Physics](./physics.md)
   - [Environmental Weapons](./environmental_weapons.md)
 - [Netcode](./netcode.md)
+- [Juice](./juice.md)
 - [Editor](./editor.md)
 - [Contributing](./contributing.md)
   - [Contributing Code](./contributing_code.md)
   - [Simple Gun Weapon](./simple_gun_weapon.md)
+- Documentation
+  - [Screen Shake](./docs/screenshake.md)
+  - [Particles](./docs/particles.md)
```

Comment:
I feel like these should be sub-sections of the "Juice" chapter? Although I can see the need to have a separate chapter focusing on implementation details, which we may want to revisit at another point.

Reply:
Oh yeah, I was going to ask about that... The "docs" are like in between code documentation and a guide, but aren't strictly related to juicing? I'm thinking we can leave it like that at the moment and find some better structure once more text drops in, it might be more apparent then.